adminSiteServer/apiRouter.ts

```diff
@@ -1967,14 +1946,14 @@ apiRouter.post(
 )
 
 // Get a list of redirects that map old slugs to charts
-apiRouter.get("/redirects.json", async (req: Request, res: Response) => ({
+apiRouter.get("/redirects.json", async (req, res) => ({
```

Comment:
Also super easy to convert to trx style; but then I also don't really get how you're choosing which routes to convert and which are staying unchanged.

Reply:
I started with roughly one entity type per PR and then upgraded the routes to the new style that were related to that entity. Over the course of the full stack most routes are converted and then the remaining (except for the ones related to the chart revision tool) I'll do in a cleanup PR at the top of the stack